simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -56,8 +58,19 @@ public StandardExports() {
   public List<MetricFamilySamples> collect() {
     List<MetricFamilySamples> mfs = new ArrayList<MetricFamilySamples>();
 
-    mfs.add(new CounterMetricFamily("process_cpu_seconds_total", "Total user and system CPU time spent in seconds.",
-        osBean.getProcessCpuTime() / NANOSECONDS_PER_SECOND));
+    try {
+      Method method = osBean.getClass().getMethod("getProcessCpuTime", null);
+      if (!method.isAccessible()) {
```

Comment:
Ah - now I follow you, because Oracle provides it through https://docs.oracle.com/javase/7/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html which was the interface used before, but now it's a concrete class, which either implements that iface, or com.ibm.lang.management.OperatingSystemMXBean (also an iface) - depending on the VM-flavour - but the common superinterface they both (IBM/Oracle) implement does not expose  getProcessCpuTime().